### ISSUE TITLE
GoBGP seems to exit over and over. 

### ISSUE THREAD

**Issue by xfy777** [rhicks/bgp-dash#12]
Mongo is saving data but the dashboard is not displaying it--

I know this hasn't been maintained for some time but it's a very cool project and I was wondering if you'd be willing to do a revamp? :DD

**Comment by yannik995** [comment]
Yes, everything was pretty outdated, I updated it a bit in a fork:
https://github.com/NetDevUG/bgp-dashboard
